components/nimbus/build.rs

```diff
@@ -2,7 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use std::{env, process::Command};
+
 pub fn main() {
     #[cfg(feature = "uniffi-bindings")]
     uniffi_build::generate_scaffolding("./src/nimbus.udl").unwrap();
+
+    // Run Glean Parser via tools/bootstrap_glean_rust.py python script
+    let out_dir = env::var("OUT_DIR").unwrap();
+    Command::new("python")
```

Comment:
Wait, this is not creating a virtual env now anymore anywhere, does it?
We could actually put the virtual env into the target directory. It's somewhere up from `OUT_DIR` I guess.

Reply:
Yes, once there are more than just Nimbus using the virtual environment, it will be good to have it shared among the other components.